.travis.yml

```diff
@@ -4,7 +4,7 @@ sudo: false
 
 env:
   global:
-    - INFECTION_FLAGS='--threads=4 --min-msi=67 --min-covered-msi=75 --coverage=coverage --log-verbosity=none'
+    - INFECTION_FLAGS='--threads=$(nproc) --min-msi=67 --min-covered-msi=75 --coverage=coverage --log-verbosity=none'
```

Comment:
I think it's better to adjust to Travis CI here instead of hardcoding the value

Reply:
It's a kind of a good idea, but really the throughput does not scale linearly (it was saturating very quick last time I checked) so we may be better off with just 4 processes, letting the CI system use the remaining resources.